integration_setup.sh

```diff
@@ -209,19 +210,32 @@ set_nextcloud_to_storage_response=$(curl -s -X POST -u${OP_ADMIN_USERNAME}:${OP_
 
 
 # if there is no error from the last api call then the integration can be declared successful
-
 response_type=$(echo $set_nextcloud_to_storage_response | jq -r "._type")
 if [ ${nextcloud_client_id} == "Error" ]; then
-	error_message=$(echo $set_nextcloud_to_storage_response | jq -r ".message")
-	log_error "${error_message}"
-	log_error "Setup of the integration failed :( !!"
+	deleteOPStorageAndPrintErrorResponse "$set_nextcloud_to_storage_response"
 	exit 1
 fi
 
 log_info "success!"
-log_success "Setup of the integration was successful :) !!"
+
 if [ ${SETUP_PROJECT_FOLDER} == true ]; then
-	log_info "OpUserAppPassword=${openproject_user_app_password}"
-	log_info "You can use this app password to make request to nextcloud as user 'OpenProject'"
+	# save the application password to Open Project
```

Comment:
```suggestion
    # save the application password to Open Project
```

The indent looks wrong here,